modules/Twitter/src/main/java/totetmatt/gephi/twitter/MainTwitterWindows.java

```diff
@@ -0,0 +1,705 @@
+
+    private void credential_goto_twitter_buttonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_credential_goto_twitter_buttonActionPerformed
+        if (Desktop.isDesktopSupported()) {
+            try {
+                Desktop.getDesktop().browse(new URI("https://apps.twitter.com/"));
+            } catch (IOException e) {
+                /* TODO: error handling */ } catch (URISyntaxException ex) {
+                Exceptions.printStackTrace(ex);
+            }
+        } else {
+            /* TODO: error handling */ }
+    }//GEN-LAST:event_credential_goto_twitter_buttonActionPerformed
+
+    private void credential_load_buttonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_credential_load_buttonActionPerformed
+        /**/
+        streamer.getCredentialProperty().load();
+        credential_consumer_key_field.setText(streamer.getCredentialProperty().getConsumerKey());
+        credential_consumer_secret_field.setText(streamer.getCredentialProperty().getConsumerSecret());
+        credential_access_token_field.setText(streamer.getCredentialProperty().getToken());
+        credential_access_token_secret_field.setText(streamer.getCredentialProperty().getTokenSecret());
+    }//GEN-LAST:event_credential_load_buttonActionPerformed
+
+    private void credential_save_buttonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_credential_save_buttonActionPerformed
+        /**/
+        streamer.getCredentialProperty().setConsumerKey(credential_consumer_key_field.getText().trim());
+        streamer.getCredentialProperty().setConsumerSecret(credential_consumer_secret_field.getText().trim());
+        streamer.getCredentialProperty().setToken(credential_access_token_field.getText().trim());
+        streamer.getCredentialProperty().setTokenSecret(credential_access_token_secret_field.getText().trim());
+        streamer.getCredentialProperty().save();
+    }//GEN-LAST:event_credential_save_buttonActionPerformed
+
+    private void load_tracking_buttonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_load_tracking_buttonActionPerformed
+        final JFileChooser fc = new JFileChooser();
+
+        FileNameExtensionFilter filter = new FileNameExtensionFilter(org.openide.util.NbBundle.getMessage(MainTwitterWindows.class, "MainTwitterWindows.file_chooser.format"), "json", "application/json");
+        fc.setFileFilter(filter);
+        int returnVal = fc.showOpenDialog(this);
+        if (returnVal == JFileChooser.APPROVE_OPTION) {
+            File file = fc.getSelectedFile();
+            try {
+                streamer.loadTracking(file);
+            } catch (IOException ex) {
+                Exceptions.printStackTrace(ex);
+            } catch (JSONException ex) {
+                Exceptions.printStackTrace(ex);
+            }
+            refreshTracking();
+        } else {
+            //log.append("Open command cancelled by user." + newline);
```

Comment:
You could either just remove this else statement (since it is empty) or consider replacing the commented out line with another logging statement:
```
Logger.getLogger("").info("Open command cancelled by user.");
````